docs/userguide/i18n.md

```diff
@@ -85,7 +85,7 @@ successful internationalization software deliverable:
 
 1.  Separate the program's executable code from its UI elements.
 
-2.  Avoid making cultural assumptions.
+1.  Avoid making cultural assumptions.
```

Comment:
not going to block on this one.  for the future i think we should setup/use a linter to enforce a consistent style.

Reply:
Thanks. I don't have strong feelings about this.